siege_utilities/geo/providers/etter_filter.py

```
@@ -0,0 +1,293 @@
+        # result.reference_location == "Lausanne"
+        # result.buffer_distance_m == 5000.0
+
+    Pass ``llm=`` to use a pre-built chat model (e.g. for tests or for
+    pinning to a specific Anthropic / Azure deployment). All other
+    constructor kwargs are forwarded to the upstream parser.
+    """
+
+    def __init__(
+        self,
+        *,
+        llm: Any = None,
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `llm`

(ANN401)